web_tree_no_open/README.rst

```diff
@@ -0,0 +1,81 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Web Tree No Open
+================
+
+This module extends the functionality of tree views.
```

Comment:
What about?
This module extends the functionality of tree views and adds a new attribute to deactivate the onclick behaviour in them.
The new configuration attribute 'open' is to be used in the same way as existing ones 'edit', 'create' and 'delete'.